lib/facter/fips_ciphers.rb

```diff
@@ -0,0 +1,9 @@
+# List available FIPS-compatible OpenSSL ciphers on the system
+# Returns: Array[String]
+Facter.add('fips_ciphers') do
+  confine :kernel => 'Linux'
+
+  setcode do
+    Facter::Core::Execution.exec(`openssl ciphers FIPS:!LOW`).split(':')
```

Comment:
This does not work on CentOS 6.  I had to change it to 'openssl ciphers FIPS:LOW'  changing the ` to ' and getting rid of the !.